packages/mailutils/build.sh

```diff
@@ -6,7 +6,9 @@ TERMUX_PKG_VERSION=3.12
 TERMUX_PKG_SRCURL=https://ftp.gnu.org/gnu/mailutils/mailutils-$TERMUX_PKG_VERSION.tar.xz
 TERMUX_PKG_SHA256=6d43fa217c4ac63f057de87890c562d170bb92bc402368b5fbc579e4c2b3a158
 TERMUX_PKG_BUILD_DEPENDS="libandroid-glob, libcrypt"
+TERMUX_PKG_BUILD_IN_SRC=true
```

Comment:
Why is this necessary? It builds fine out-of-src for me

Reply:
I adding this because building result in an error of file not found and the find found in src folder, I cant reproduce this behavior now so i'll revert.